README.md

```diff
@@ -212,69 +219,142 @@ module.exports = {
 | microsoftedge | 17.17134 | requires [Sauce Labs](https://screener.io/v2/docs/sauce) Integration |
 | safari | 11.1 | requires [Sauce Labs](https://screener.io/v2/docs/sauce) Integration |
 
+### Sauce Connect Integration
+
+When using Sauce Labs browsers, you have the option to use the Sauce Connect tunnel by setting the flag `launchSauceConnect: true`. When enabled, Sauce Connect will be launched and managed by this module, and assigned a unique tunnel identifier.
 
+  ```javascript
+  sauce: {
+    username: 'sauce_user',
+    accessKey: 'sauce_access_key',
+    maxConcurrent: 10, // optional available concurrency you have from Sauce Labs
+    launchSauceConnect: true // optional,
+  }
+  ```
 
+#### Sauce Connect Notes
 
+- Must use Sauce browsers
 
-### Additional Configuration Options
+  When you run your tests in sauce labs, you could only run your tests inside sauce browsers, which means that you have to specify both the name of the browser and the version of the browser up to one digit after the decimal dot:
+
+    ```javascript
+      browsers: [
+        {
+          browserName: 'chrome',
+          version: '78.0'
+        },
+        {
+          browserName: 'firefox',
+          version: '70.0'
+        },
+        {
+          browserName: 'internet explorer',
+          version: '11.0'
+        }
+      ]
+    ```
+
+  If you don't follow the rules, the runner will throw you the error: `Only Sauce Labs browsers with version can be used when launchSauceConnect flag is enabled`.
+
+- Where Sauce Connect logs are stored
+
+  The sauce connect tunnel will generate `sauce-connect.log` log file for debugging purpose at the root directory of your project.
+
+- Sauce tunnel can not be used together with ngrok tunnel
+
+  `ngrok` tunnel is forbidden when you run the sauce connect tunnel, which means you can not set
+
+    ```javascript
+      tunnel: {
+        host: 'localhost:3000'
+      }
+    ```
+
+  when you enable `launchSauceConnect`. If you don't follow the rules, the runner will throw you the error: `tunnel option cannot be set when launchSauceConnect flag is enabled`.
+
+- `launchSauceConnect` and `tunnelIdentifier`
+
+  Please note that if you turn `launchSauceConnect` to be true, the runner will set a unique `tunnelIdentifier` for you insead of setting by yourself. If you don't follow the rules, the runner will throw you the error: `tunnelIdentifier cannot be set when launchSauceConnect flag is enabled`.
+
+- Sauce Connect valid ports
+
+  When you run your tests in your localhost, please be cautious that sauce connect only supports these [valid ports](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+FAQS#SauceConnectProxyFAQs-CanIAccessApplicationsonlocalhost?).
+
+- Further question
```

Comment:
Please change to: "For additional information on Sauce Connect please refer to the Sauce Connect FAQ and Sauce Connect Troubleshooting documentation."

Reply:
changed